content/en/tracing/trace_collection/automatic_instrumentation/single-step-apm/linux.md

```diff
@@ -29,11 +29,11 @@ To enable APM on a Linux host:
 
 ## Set SDK tracer versions
 
-By default, Single Step Instrumentation installs the latest major versions of Datadog APM SDKs. Minor version updates are applied automatically when they become available.
+By default, Single Step Instrumentation installs the latest versions of Datadog APM SDKs.
 
-You may want to customize SDK versions based on your application's language version or specific environment requirements. You can control the major and minor versions used by customizing library versions during setup.
+You may want to choose specific SDK versions.
```

Comment:
What was your reasoning behind making this change? I think the information provided by the previous version is relevant and worth including.

Reply:
I've reinstated the reasons part - when I read it I thought it was implying that you could customize versions depending on the language version, but having re-read it I think it's giving reasons why you want to choose a specific version.

I'm no sure the second sentence is adding anything, but I can restore it if you think it's useful?

Reply:
Thanks, the latest version looks reasonable.